modules/Bio/EnsEMBL/DBSQL/DBConnection.pm

```diff
@@ -722,6 +722,13 @@ sub prepare {
   Description: Prepares a SQL statement using the internal DBI database handle
                and returns the DBI statement handle. The prepared statement is
                cached so that it does not have to be prepared again.
+
+               When using this function it is important to ensure either
+               completel retrieval of all information from the request when
```

Comment:
"That's a misprint, innit?"</round_the_horne>

Reply:
This could be changed to something like:

"If not retrieving all rows then `finish()` should be called on the object to free up the statement handle."

Reply:
So close... come on Mark, you can do it